test/fixtures/archived_patients.yml

```diff
@@ -0,0 +1,7 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```

Comment:
We generally use factories (via the FactoryBot gem) rather than fixtures, so you can drop this file. Check out `test/factories/patient.rb` for an example of what you can use instead.

Reply:
Thanks! I haven't even looked at the test files yet, but I'll be pinging you when I do, since I haven't built one from scratch with FactoryBot before!